src/app/pages/home-teacher/home-teacher.component.ts

```diff
@@ -1,12 +1,40 @@
-import { Component } from '@angular/core';
+import { Component, inject } from '@angular/core';
+import { StudentsService } from '../../services/students.service';
+import { IStudentInfoInterface } from '../../interfaces/iStudentInfo.interface';
+import { AuthService } from '../../services/auth.service';
+import { TeachersService } from '../../services/teachers.service';
+import { ITeacherInfoInterface } from '../../interfaces/iTeacherInfoInterface';
+import { IListResponseInterface } from '../../interfaces/iListResponse.interface';
+import { StudentsComponent } from '../students/students.component';
 
 @Component({
   selector: 'app-home-teacher',
   standalone: true,
-  imports: [],
+  imports: [StudentsComponent],
   templateUrl: './home-teacher.component.html',
   styleUrl: './home-teacher.component.css'
 })
 export class HomeTeacherComponent {
 
+  studentsService = inject(StudentsService);
+  teachersService = inject(TeachersService);
+  students: IStudentInfoInterface[] = [];
+  teacher: ITeacherInfoInterface[] = []
+  iList: IListResponseInterface = { total: 0, data: [] };
```

Comment:
no se está usando, borrar

Reply:
Borrado todo menos Teacher service que se esta usando en ngOnInit, en la linea :
const resp = this.teachersService.getTeacherInfo(this.teacherId);